services/gitdiff/gitdiff.go

```diff
@@ -718,6 +718,14 @@ func ParsePatch(maxLines, maxLineCharacters, maxFiles int, reader io.Reader) (*D
 
 			}
 
+			if len(diff.Files) >= maxFiles {
```

Comment:
Seems like this could be moved further up

Reply:
Which line did you think is best?

Reply:
@lunny line 688